### PR TITLE
Update to Diagrams in developer guide

### PR DESCRIPTION
**Pull request by josephambe**
Updated Sequence Diagrams (x2), Architecture Diagram and LogicComponentClassDiagram.png

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -136,23 +136,23 @@ Each of the four components
 For example, the `Logic` component (see the class diagram given below) defines it's API in the `Logic.java` interface and exposes its functionality using the `LogicManager.java` class.
 
 .Class Diagram of the Logic Component
-image::LogicClassDiagram.png[width="800"]
+image::[width="800"]
```
Comment (jeremiah-ang) [inline docs/DeveloperGuide.adoc]: Do you mean to remove this image? 
```suggestion

```

Comment (josephambe) [reply docs/DeveloperGuide.adoc]: Should be good now

Comment (josephambe) [reply docs/DeveloperGuide.adoc]: And from what I can see, the current Developer Guide even without this merge already has this diagram updated from another PR. So should be fine either way

Comment (jeremiah-ang) [reply docs/DeveloperGuide.adoc]: As in, this change will remove the image. Is this what you intend to do?

Comment (josephambe) [reply docs/DeveloperGuide.adoc]: Okay, NOW it should be good

**Pull request merged**